packages/shared/src/logger.ts

```diff
@@ -30,9 +30,29 @@ function anonymizeStackTrace(stack: string): string {
     .join("\n");
 }
 
-type LogLevel = "error" | "warn" | "info" | "debug";
+export function formatTags(tags?: Tags) {
```

Comment:
If this is going to be used by multiple things (logger aka Grafana _and_ Sentry) maybe it makes sense to put it in its own standalone module? (So the scope is clearer?)